SubmittedAssignments/Swapnil_Patil_05/script.js

```
@@ -0,0 +1,237 @@
+      //content section 1
+      const objective = document.createElement('div');
+      objective.classList.add(config.objectiveClass);
+      background.appendChild(objective);
+      
+      var career = document.createElement('h1');
+      career.classList.add(config.careerClass);
+      career.textContent = (data.career);
+      objective.appendChild(career);
+      
+      var hr = document.createElement('hr');
+      hr.classList.add(config.hrClass);
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)